nk/main.go

```diff
@@ -32,16 +31,35 @@ import (
 // this will be set during compilation when a release is made on tools
 var Version string
 
+const defaultVanMax = 10_000_000
+
 func usage() {
-	log.Fatalf("Usage: nk [-v] [-gen type] [-sign file] [-verify file] [-inkey keyfile] [-pubin keyfile] [-sigfile file] [-pubout] [-e entropy] [-pre vanity]\n")
+	log.Fatalf(`Usage: nk [options]
+    -v                    Show version
+    -gen <type>           Generate key for <type>, e.g. nk -gen user
```

Comment:
Would be really helpful to have a list of valid types instead of having to try it and fail.  Could refer to a const string which is defined lower-down, by where you emit it for the error case, instead of duplicating the list in code.